client/app/bundles/course/leaderboard/components/tables/LeaderboardTable.tsx

```diff
@@ -329,6 +329,7 @@ const LeaderboardTable: FC<Props> = (props: Props) => {
   );
 };
 
+// eslint-disable-next-line react/display-name
```

Comment:
This is still here cos even after globally disabling the error it still shows up for this particular file